example/main.go

```diff
@@ -18,6 +19,7 @@ func main() {
 			"app":         "example",
 			"environment": "development",
 		})
+		// .WithBasicAuth("admin", "secretpassword")
```

Comment:
```suggestion
		.WithBasicAuth("admin", "secretpassword") // Optional
```
Should also reflect this in the readme